TypeScript.py

```diff
@@ -281,6 +281,9 @@ def __init__(self):
         self.fileMap = {}
         self.refInfo = None
         self.versionST2 = False
+        self.tempFileMap = {}
+        self.tempFileList = []
```

Comment:
Is the mapping between seq number and temp file name used somewhere? Or is it more like a collection of "temp files used but not available now"?